### ISSUE TITLE
chatgpt-telegram.exe window disappears without error message

### ISSUE THREAD

**Issue by Tegasus9** [m1guelpf/chatgpt-telegram#24]
I recently installed chatgpt-telegram.exe on my Windows10 system computer and was able to successfully sign in OpenAI website. However, after a few seconds the window disappears suddenly without any error message. I'm not sure what's causing this issue or how to fix it.

Also, my .env is configured and get my TELEGRAM_TOKEN writed in it.And I tried run this program with 'Run as administrator'.

Can anyone provide some guidance on what might be happening and how I can troubleshoot this problem?

Thanks in advance for your help.

**Comment by m1guelpf** [comment]
Did you create the `.env` file as instructed in the README?

**Comment by Tegasus9** [comment]
> Did you create the `.env` file as instructed in the README?

OMG……Thank you for updating the README this morning. I didn't know until today that I could use WindowsPowerShell to open `.exe` program. After running, the friendly reminder told me "Couldn't start Telegram bot: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond." When I enabled the proxy and restarted the program, everything worked normally.

Finally I saw "Started Telegram bot! Message @**_bot to start.".Thank you.
